adagrams/game.py

```diff
@@ -1,11 +1,162 @@
+#Maggie T adagrams project 
+# Billie Betts adagram project
+
+import string
+import random
+
+quantity_data = {1: ['Z', 'X', 'K', 'Q', 'J'],
+                  2: ['B','C','F','H','M', 'P','V','W','Y'],
+                  3: ['G'],
+                  4: ['D', 'L', 'S', 'U'],
+                  6: ['N', 'R', 'T'],
+                  8: ['O'],
+                  9: ['A', 'I'],
+                  12: ['E']}
+
+value_data = {1: ['A', 'E', 'I', 'O', 'U', 'L', 'N', 'R', 'S', 'T'],
+              2: ['D','G'],
+              3: ['B', 'C', 'M', 'P'],
+              4: ['F', 'H', 'V', 'W', 'Y'],
+              5: ['K'],
+              8: ['J', 'X'],
+              10: ['Q', 'Z']}
+
+#Takes the two inputs and stores them in one dictionary 
+def generate_alphabet_data_list(quantity_data, value_data):
```

Comment:
Nice work using a custom helper function. I see that you are creating a new dictionary for the data above but why not create the data structure from scratch? Also why are we combining the data sets? We would want to handle the data separately.